<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signCutter</class>
 <widget class="QDialog" name="signCutter">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>810</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777213</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sign Cutter -- 赵鹏程组【郭琛、廖岩岩、刘璇、唐志雄、王静怡、张媛、赵鹏程（自然排序）】@空间数据智能计算</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="layoutWidget">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>导入源影像</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QToolButton" name="resultLocationToolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>切片　输出路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="resultLocationLineEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="resultLocationToolButton_pos">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>正样本输出路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="resultLocationLineEdit_pos">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="resultLocationToolButton_neg">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>负样本输出路径</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="resultLocationLineEdit_neg">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="sourceGroupBox_inputs">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>源影像</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QListView" name="sourceListView">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="selectedGroupBox_cutouts">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>路牌切片</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QListView" name="croppedObjectImageListView">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="selectedGroupBox_pos">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>正样本（+）</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QListView" name="positiveSign">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="selectedGroupBox_neg">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>负样本（-）</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QListView" name="negativeSign">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="beginingButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>|&lt;&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="preFrameButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextFrameButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="endButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&gt;&gt;|</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="zoomComboBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="zoomOutToolButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="zoomInToolButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="selectButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>保存切片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_msg">
             <property name="text">
              <string>-null-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>删除切片</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="SelectionView" name="workspaceGraphicsView">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGraphicsView" name="thumbnailGraphicsView">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_helpgen">
               <property name="text">
                <string>查看切片记录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_genpos">
               <property name="text">
                <string>生成正样本</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_genneg">
               <property name="text">
                <string>生成负样本</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGraphicsView" name="realTimeCroppedImageGraphicsView">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectionView</class>
   <extends>QGraphicsView</extends>
   <header location="global">SelectionView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sourceListView</tabstop>
  <tabstop>resultLocationToolButton</tabstop>
  <tabstop>beginingButton</tabstop>
  <tabstop>preFrameButton</tabstop>
  <tabstop>nextFrameButton</tabstop>
  <tabstop>endButton</tabstop>
  <tabstop>zoomComboBox</tabstop>
  <tabstop>zoomOutToolButton</tabstop>
  <tabstop>zoomInToolButton</tabstop>
  <tabstop>workspaceGraphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
